sdk/tests/conformance2/buffers/uniform-buffers.html

```diff
@@ -127,8 +148,15 @@
 
     wtu.setupUnitQuad(gl);
 
+    var program = wtu.setupProgram(gl, ['vshader', 'fbadshader']);
+    if (!program) {
+        testPassed("To define the same uniform in two uniform blocks should fail");
+    }
+
     var program = wtu.setupProgram(gl, ['vshader', 'fshader']);
-    wtu.glErrorShouldBe(gl, gl.NO_ERROR, "should be able to compile shader with uniform blocks without error");
```

Comment:
I added the new uniform blocks in fbadshader, used glErrorShouldBe( ) to test the compile/link status, found it succeed to compile in Chromium. It was a bug, I thought. However, I found that the JS code was wrong: if shaders fail to compile or link, it will not generate glError. As a result, using wtu.glErrorShouldBe( ) to detect setupProgram( ) is not appropriate. 

In fact, setupProgram will detect the COMPILE_STATUS and LINK_STATUS and return NULL if the shaders are failed to compile/link.